Blank screen (404) on Ubuntu 22.04

**Describe the bug**
Version 1.3.14 doesn't work on Ubuntu 22.04, the screen just stays blank and the console shows:
```
$ teams-for-linux 
[ERROR] Server returned error code '404'
[2071771:1018/094436.347976:ERROR:nss_util.cc(357)] After loading Root Certs, loaded==false: NSS error code: -8018
```

I also tried clearing the cache and removing the config folder but nothing helps.
Also, `teams-for-linux --webDebug true` doesn't do anything.

**To Reproduce**
Steps to reproduce the behavior:
1. Run the app
2. See error

**Screenshots**
![image](https://github.com/IsmaelMartinez/teams-for-linux/assets/4000539/d46662e3-d360-4bf3-ba7f-0097bc4da2c9)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.3
 - Installation package: deb
 - Version: 1.3.14

--- a/app/config/README.md
+++ b/app/config/README.md
@@ -33,6 +33,7 @@ Here is the list of available arguments and its usage:
 | config | config file location | ~/.config/teams-for-linux/config.json |
 | defaultURLHandler | Default application to be used to open the HTTP URLs |  |
 | disableAutogain | A flag indicates whether to disable mic auto gain or not | false |
+| disableGpu | A flag to disable GPU and hardware acceleration (can be useful if the window remains blank) | false |
 | disableMeetingNotifications | Whether to disable meeting notifications or not | false |
 | disableNotifications | A flag to disable all notifications | false |
 | disableNotificationSound | Disable chat/meeting start notification sound | false |

--- a/app/config/index.js
+++ b/app/config/index.js
@@ -136,6 +136,11 @@ function argv(configPath) {
 				describe: 'A flag indicates whether to disable mic auto gain or not',
 				type: 'boolean'
 			},
+			disableGpu: {
+				default: false,
+				describe: 'A flag to disable GPU and hardware acceleration (can be useful if the window remains blank)',
+				type: 'boolean'
+			},
 			disableMeetingNotifications: {
 				default: false,
 				describe: 'Whether to disable meeting notifications or not',

--- a/app/index.js
+++ b/app/index.js
@@ -76,6 +76,16 @@ if (!app.isDefaultProtocolClient(protocolClient, process.execPath)) {
 
 app.allowRendererProcessReuse = false;
 
+if (config.disableGpu) {
+	logger.info('Disabling GPU support...');
+	app.commandLine.appendSwitch('disable-gpu');
+	app.commandLine.appendSwitch('disable-software-rasterizer');
+	// app.commandLine.appendSwitch('disable-gpu-compositing');
+	// app.commandLine.appendSwitch('disable-gpu-rasterization');
+	// app.commandLine.appendSwitch('disable-gpu-sandbox');
+	// app.disableHardwareAcceleration();
+}
+
 if (!gotTheLock) {
 	logger.info('App already running');
 	app.quit();

--- a/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
+++ b/com.github.IsmaelMartinez.teams_for_linux.appdata.xml
@@ -14,6 +14,14 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="1.3.16" date="2023-10-30">
+			<description>
+				<ul>
+					<li>New: add flag --disableGpu to disable GPU and hardware acceleration</li>
+					<li>Update: Electron version upgraded to 27.0.2</li>
+				</ul>
+			</description>
+		</release>
 		<release version="1.3.13" date="2023-09-29">
 			<description>
 				<ul>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.14",
+  "version": "1.3.16",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "yargs": "17.7.2"
   },
   "devDependencies": {
-    "electron": "^27.0.0",
+    "electron": "27.0.2",
     "electron-builder": "24.6.4",
     "eslint": "8.51.0",
     "yarn": "1.22.19"

